extensions/rocksdb-repos/ProvenanceRepository.cpp

```diff
@@ -145,9 +112,12 @@ void ProvenanceRepository::destroy() {
 }
 
 uint64_t ProvenanceRepository::getKeyCount() const {
+  auto opendb = db_->open();
+  if (!opendb) {
+    return 0;
+  }
   std::string key_count;
-  db_->GetProperty("rocksdb.estimate-num-keys", &key_count);
-
+  opendb->GetProperty("rocksdb.estimate-num-keys", &key_count);
```

Comment:
is this different from `getRepositoryEntryCount`?

Reply:
Good point, as it was only used in tests, removed this in 2c1c6841b84cd569185323665d9d927a029396e9